client/static/extracting-pokemon-files.html

```diff
@@ -83,6 +83,61 @@ <h1 class="md-display-2">Extracting Pokémon from your game</h1>
         <td>KeySAV2 method is preferred</td>
       </tr>
     </table>
+    
+    <h2 class="md-headline">Sun, Moon</h2>
+    
+    <p>
+      The most common methods for obtaining the pk7 files of your Pokémon are:<br>
+    <ul>
+      <li>Reading Battle Videos</li>
+      <li>Running the homebrew application JK’s Save Manager</li>
+      <li>Reading digital save files through KeySAVe (Windows, Mac OS, Linux)</li>
+      <li>TEA checking with KeySAVe (Windows, Mac OS, Linux)</li>
+    </ul>
+    
+    <p>Here is a handy table that lists the requirement for each of the methods above:</p>
+
+    <table>
+      <tr>
+        <th>#</th>
+        <th>Method</th>
+        <th>Game version</th>
+        <th>Required hardware</th>
+        <th>Specific firmware?</th>
+        <th>Notes</th>
+      </tr>
+      <tr>
+        <td>1</td>
+        <td><a href="/#/how-to-pk7-1-bvs">Battle Videos</a></td>
+        <td>Digital/retail</td>
+        <td>Internet access or another console, SD card reader</td>
+        <td>No</td>
+        <td><b>New to this? Start here!</b></td>
+      </tr>
+      <tr>
+        <td>2</td>
+        <td><a href="/#/how-to-pk7-2-homebrew">Homebrew</a></td>
+        <td>Digital/retail</td>
+        <td>SD card reader, Nintendo 3DS Sound, Cubic Ninja (optional), Zelda OoT3D (optional), Freakyforms Deluxe (optional)</td>
+        <td>=<11.2.0</td>
+        <td>Required hardware depends on firmware version</td>
+      </tr>
+      <tr>
+        <td>3</td>
+        <td><a href="/#/how-to-pk7-3-digital-save-files">Digital save files with KeySAVe</a></td>
+        <td>Digital</td>
+        <td>SD card reader</td>
+        <td>No</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>4</td>
+        <td><a href="/#/how-to-pk7-4-tea">TEA checking with KeySAVe</a></td>
+        <td>Digital/retail</td>
+        <td>CFW, NTR CFW</td>
+        <td>=<11.2.0</td>
```

Comment:
I think this is malformed html -- to use a `<` symbol in an html page, you should use `&lt;`.